libcrux-ml-kem/proofs/verification_status.md

```diff
@@ -0,0 +1,42 @@
+# ML-KEM Verification Status
+
+This file keeps track of the current verification status of the modules in the ML-KEM implementation.
+
```

Comment:
Let's make this into a table. Happy to do something to help. But the list is hard to read.

Reply:
done.